DevEdu.API/DevEdu.DAL/Repositories/CommentRepository.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using Dapper;
+using DevEdu.DAL.Models;
+
+namespace DevEdu.DAL.Repositories
+{
+    public class CommentRepository : BaseRepository, ICommentRepository
+    {
+        public int AddComment(CommentDto commentDto)
+        {
+            return _connection.QuerySingle<int>(
+                "dbo.Comment_Insert",
```

Comment:
each name of stored procedures must be placed somewhere as a constant
otherwise, it's an example of the "magic numbers" antipattern